### PR TITLE
OSDOCS-8515-post-slideshow: Added more 4.15 Welcome Page entries

### PR DESCRIPTION
**Pull request by dfitzmau**
Version(s):
4.15

Issue: 
[OSDOCS-8515](https://issues.redhat.com/browse/OSDOCS-8515)

Link to docs preview:
[Welcome page](https://71731--ocpdocs-pr.netlify.app/openshift-enterprise/latest/welcome/)

### CODE REVIEW

**Review by shreyasiddhartha — APPROVED** [review]
Power Monitoring line LGTM!

**Comment by dfitzmau** [comment]
/label peer-review-needed

PR cannot be merged until https://github.com/openshift/openshift-docs/pull/70025 is merged. I commented out the link for now on #71731 . Requesting a peer review for now.

**Comment by cbippley** [comment]
> /label peer-review-needed
> 
> PR cannot be merged until #70025 is merged. I commented out the link for now on #71731 . Requesting a peer review for now.

The #70025 PR has been merged.

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by GroceryBoyJr** [comment]
@dfitzmau the lack of an xref in the title "Install a cluster or Oracle Cloud Infrastructure" looks odd, but it also appears to be intentional, right?

**Comment by dfitzmau** [comment]
> @dfitzmau the lack of an xref in the title "Install a cluster or Oracle Cloud Infrastructure" looks odd, but it also appears to be intentional, right?

Yes, that's correct! Thanks.

File: welcome/index.adoc [hunk 1]
```diff
@@ -101,10 +101,8 @@ ifndef::openshift-origin[]
...
 endif::openshift-origin[]
-////
-// Addition will be tracked through https://issues.redhat.com/browse/OSDOCS-9514
-- ** Install a cluster on {oci-first}**: You can use the Assisted Installer or the Agent-based Installer to install a cluster on {oci}. This means that you can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments.
-////
+
+- **Install a cluster on {oci-first}**: You can use the Assisted Installer or the Agent-based Installer to install a cluster on {oci}. This means that you can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments. See xref:../installing/installing_oci/installing-oci-assisted-installer.adoc[Using the Assisted Installer to install a cluster on {oci}] and xref:../installing/installing_oci/installing-oci-agent-based-installer.adoc[Using the Agent-based Installer to install a cluster on {oci}].
```
Comment (kcarmichael08) [inline welcome/index.adoc:105]: Not sure if the exact title of the second link needs to match the actual doc title but they're slightly different. I'll go ahead and merge and you can create a new PR if should be changed.

**Pull request merged**